packages/apputils/src/mainareawidget.ts

```diff
@@ -44,10 +44,10 @@ export class MainAreaWidget<T extends Widget = Widget>
     const trans = (options.translator || nullTranslator).load('jupyterlab');
     const content = (this._content = options.content);
     content.node.setAttribute('role', 'region');
-    content.node.setAttribute('aria-label', trans.__('notebook content'));
+    content.node.setAttribute('aria-label', trans.__('launcher content'));
```

Comment:
I don't think this is always the Launcher just as it's not always a notebook. 

I think there may be an issue about this somewhere. The aria-label probably needs to be set programmatically when a new MainAreaWidget is instantiated. 

Perhaps as a stop-gap measure, we could set these labels to:

- `main panel content`
- `main panel toolbar`